README.md

```diff
@@ -6,7 +6,6 @@ LATEST STABLE RELEASE v0.15 IS IN BRANCH [0.1x](https://github.com/lostisland/fa
 # Faraday
 
 [![Gem Version](https://badge.fury.io/rb/faraday.svg)](https://rubygems.org/gems/faraday)
-[![Build Status](https://travis-ci.org/lostisland/faraday.svg)](https://travis-ci.org/lostisland/faraday)
```

Comment:
Is there an equivalent badge for CircleCI?

Reply:
Yes, but it must be done in the configuration for the official repository.